test/config/config.go

```diff
@@ -32,4 +33,6 @@ func (c *CiliumTestConfigType) ParseFlags() {
 		"Provision Vagrant boxes and Cilium before running test")
 	flag.BoolVar(&c.HoldEnvironment, "cilium.holdEnvironment", false,
 		"On failure, hold the environment in its current state")
+	flag.StringVar(&c.SSHConfig, "cilium.SSHConfig", "",
+		"Command to get a different a different SSHConfig")
```

Comment:
Can we expand the description to state the expected format? ie,

"Specify a custom command to fetch SSH configuration (eg: 'vagrant ssh-config')"

Would be nice to add this to the contributing docs too.